pkg/storaged/utils.js

```diff
@@ -843,13 +873,28 @@
             });
         }
 
+        // HACK: get_active_usage is used for mounting and formatting so we use the absence of the subvol argument
+        // to figure out that we want to format this device.
+        // This is separate from the if's below as we also always have to umount the filesystem.
+        if (btrfs_volume && !subvol) {
+            usage.push({
+                level,
+                usage: 'btrfs-device',
+                block,
+                btrfs_volume,
+                location: btrfs_volume.data.label || btrfs_volume.data.uuid,
+                actions: get_actions(_("remove from btrfs volume")),
+                blocking: true
```

Comment:
These 9 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19690-20231221-103901-225b81f5-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)